beluga_amcl/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Comment:
@hidmic If you don't want to be listed as a maintainer, I think you should at least be listed as one of the authors on some of the packages in this repository. We can sort that out in a separate PR though :smiley:.